test/rekt/integrationsource_test.go

```diff
@@ -0,0 +1,78 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rekt
+
+import (
+	"testing"
+
+	"knative.dev/pkg/system"
+	"knative.dev/reconciler-test/pkg/environment"
+	"knative.dev/reconciler-test/pkg/eventshub"
+	"knative.dev/reconciler-test/pkg/k8s"
+	"knative.dev/reconciler-test/pkg/knative"
+
+	"knative.dev/eventing/test/rekt/features/integrationsource"
+)
+
+func TestIntegrationSourceWithSinkRef(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
+		environment.Managed(t),
+	)
+	t.Cleanup(env.Finish)
+
+	env.Test(ctx, t, integrationsource.SendsEventsWithSinkRef())
+}
+
+func TestIntegrationSourceWithTLS(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
+		environment.Managed(t),
+		eventshub.WithTLS(t),
+	)
+
+	env.ParallelTest(ctx, t, integrationsource.SendEventsWithTLSRecieverAsSink())
+	env.ParallelTest(ctx, t, integrationsource.SendEventsWithTLSRecieverAsSinkTrustBundle())
+}
+
+//func TestIntegrationSourceSendsEventsWithOIDC(t *testing.T) {
+//	t.Parallel()
+//
+//	ctx, env := global.Environment(
+//		knative.WithKnativeNamespace(system.Namespace()),
+//		knative.WithLoggingConfig,
+//		knative.WithTracingConfig,
+//		k8s.WithEventListener,
+//		environment.Managed(t),
+//		eventshub.WithTLS(t),
+//	)
+//
+//	env.Test(ctx, t, integrationsource.SendsEventsWithSinkRefOIDC())
+//}
```

Comment:
can we drop this?

Reply:
yeah. we can drop this - for now as the DP is not yet ready for this